cards_3.c

```diff
@@ -0,0 +1,51 @@
+#include <stdio.h>
+#include <time.h>
```

Comment:
I got this as output:
```
32769 of 
9 of clubs
32769 of clubs
5 of spades
4 of clubs
```
not sure where you went wrong - you should have included comments so I can navigate your code!